superset-frontend/plugins/plugin-chart-pivot-table/src/react-pivottable/TableRenderers.jsx

```diff
@@ -24,7 +24,11 @@ import { PivotData, flatKey } from './utilities';
 import { Styles } from './Styles';
 
 const parseLabel = value => {
```

Comment:
We do not need to change the parseLabel function. Instead, we can simply translate the METRIC_KEY constant in superset-frontend/plugins/plugin-chart-pivot-table/src/PivotTableChart.tsx:58